src/documentation/fares_internal_format.md

```diff
@@ -0,0 +1,35 @@
+# Representing fares in Navitia Transit Model
+## Overview
+This document describes how fare structures are modeled in Navitia Transit Model. The internal model is composed of the followning objects:
+- Ticket
+- Rules
+
+## Model details
+### Ticket
+This object contains the necessary information to specify a ticket.
+
+Property | Type | Required | Description
+--- | --- | --- | ---
+id | UUID | Yes | Unique identifier
+input_data_format | Enum | Yes | Source of the fare data, fixed value `netex_fares_nl`.
+input_ticket_id | String | No | Id of the ticket found in the source data.
+start_date | Date | Yes | Start date for the validity period of the ticket price.
+end_date | Date | Yes | End date for the validity period of the ticket price. This date is included in the validity period interval.
```

Comment:
Why that restriction on end_date being into validity_period?
For idfm it can be a burden, no?

Reply:
I leave it as is for now, because it's simpler to begin with.